Add Cube Stack Umbrella Helm Chart

adnanrahic opened the pull request:
**Check List**
- [ ] Tests has been run in packages where changes made if available
- [ ] Linter has been run for changed code
- [ ] Tests for the changes have been added if not covered yet
- [ ] Docs have been added / updated if required

**Issue Reference this PR resolves**

https://github.com/cube-js/cube.js/issues/3337

hassankhan committed:
docs(caching): document how to inspect pre-aggregations in Cube Store with MySQL

Commit:
Added cubestack umbrella chart

Commit:
Added cubestack umbrella chart

Commit:
Updated values

Commit:
Updated values

Commit:
Added externalDefault global config

Commit:
Added externalDefault global config

Commit:
Added values for cubestack

Commit:
Added values for cubestack

Commit:
Added default redis url and pass

Commit:
Added default redis url and pass

Commit:
Update readme

Commit:
Update readme

Commit:
Update readme

Commit:
Update readme

lvauvillier commented on examples/helm-charts/cubejs/README.md:
```diff
@@ -179,6 +179,7 @@ cubestore:
 | `global.devMode`                   | If true, enables development mode                                                                                               | `false` |
 | `global.debug`                     | If true, enables debug logging                                                                                                  | `false` |
 | `global.logLevel`                  | The logging level for Cube.js                                                                                                   | `warn`  |
+| `global.externalDefault`           | If true, uses Cube Store for storing Pre-aggregations                                                                           | `true`  |
```
I'm not sure this is only for cubestore

adnanrahic replied:
Nice catch! I'll edit this to mention other databases too.

lvauvillier commented on examples/helm-charts/cubestack/README.md:
```diff
@@ -0,0 +1,76 @@
+  redis:
+    ## The host URL for a Redis server
+    ## Naming this release "cubestack" will give you this default Redis URL 
+    url: redis://cubestack-redis-master:6379
```
Maybe we can automatically configure this value if `redis.enabled` = true

adnanrahic replied:
Do you have an idea on how to do this?

lvauvillier commented on examples/helm-charts/cubestack/values.yaml:
```diff
@@ -0,0 +1,810 @@
+#### Cubejs ####
+################
+cubejs:
+  enabled: true
```
I think there is no need to copy all values here. You can just keep the `xxx.enabled` values

ilya-biryukov committed:
chore(cubestore): add option to build docker image without AVX2

By default, we still build with AVX2.
Use `docker --build-arg WITH_AVX2=0` to build without AVX2.

ilya-biryukov committed:
chore(cubestore): updates to rolling window (#3347)

- fix `GROUP BY DIMENSION` with timestamps
 - add `OFFSET` clause for rolling aggregates

See DataFusion commits for more details.

paveltiunov committed:
feat: Mixed rolling window and regular measure queries from rollup support (#3326)

* feat: Mixed rolling window and regular measure queries from rollup support

* chore: Mixed rolling window and regular measure queries from rollup support -- revert runningTotal support as it can't be coerced to rollingWindow

* chore: Measure mixing cases fixing

* chore: Add offset support

* chore(cubestore): Week interval support

omab committed:
feat: Support multi-value filtering on same column through FILTER_PARAMS (#2854) Thanks to @omab!

* Support multi-value filtering on same column

* Test multivalue filter on same column

* Simplify filter length check

paveltiunov committed:
v0.28.29

hassankhan committed:
docs(deployment): add Kubernetes to Overview page and fix menu order

ilya-biryukov committed:
fix(cubestore): fix crash on 'unexpected accumulator state List([NULL])'

hassankhan committed:
docs(schema): add blurb about `refreshKey` being evaluated on a per-partition basis (#3356)

keydunov committed:
docs: Update Using Pre-Aggs -> Partitioning (#3305)

* docs: Update Using Pre-Aggs -> Partitioning

* Update docs/content/Caching/Using-Pre-Aggregations.md

Co-authored-by: Ray Paik <ray@cube.dev>

* Update docs/content/Caching/Using-Pre-Aggregations.md

Co-authored-by: Ray Paik <ray@cube.dev>

* Update docs/content/Caching/Using-Pre-Aggregations.md

Co-authored-by: Hassan Khan <hassan@cube.dev>

Co-authored-by: Ray Paik <ray@cube.dev>
Co-authored-by: Hassan Khan <hassan@cube.dev>

vasilev-alex committed:
fix(playground): sql formatting

vasilev-alex committed:
chore: rollup designer redesign (#3360)

* chore: rollup designer redisign

* style fixes

* modal is closed by default

vasilev-alex committed:
fix(playground): warn when a query can't be rolled up

vasilev-alex committed:
chore: style fix

vasilev-alex committed:
v0.28.30

rchkv committed:
docs(recipes): Getting unique values for a field (#3359)

igorlukanin committed:
docs(recipes): EAV (#3300)

* Recipe

* Update docs/content/Examples-Tutorials-Recipes/Entity-Attribute-Value.md

Co-authored-by: Adnan Rahić <ado.raha198@gmail.com>

* Update docs/content/Examples-Tutorials-Recipes/Entity-Attribute-Value.md

Co-authored-by: Adnan Rahić <ado.raha198@gmail.com>

* Update docs/content/Examples-Tutorials-Recipes/Entity-Attribute-Value.md

* Update docs/content/Examples-Tutorials-Recipes/Entity-Attribute-Value.md

Co-authored-by: Hassan Khan <hassan@cube.dev>

* Update docs/content/Examples-Tutorials-Recipes/Entity-Attribute-Value.md

Co-authored-by: Ray Paik <ray@cube.dev>

* Update docs/content/Examples-Tutorials-Recipes/Entity-Attribute-Value.md

Co-authored-by: Ray Paik <ray@cube.dev>

* Fix

* Update docs/content/Examples-Tutorials-Recipes/Entity-Attribute-Value.md

Co-authored-by: Hassan Khan <hassan@cube.dev>

* Update docs/content/Examples-Tutorials-Recipes/Entity-Attribute-Value.md

Co-authored-by: Hassan Khan <hassan@cube.dev>

* Line breaks

* Complete the text and example

* format document

Co-authored-by: Adnan Rahić <ado.raha198@gmail.com>
Co-authored-by: Anton Rychkov <antrychkov@gmail.com>
Co-authored-by: Hassan Khan <hassan@cube.dev>
Co-authored-by: Ray Paik <ray@cube.dev>

vasilev-alex committed:
fix(playground): unsupported regex in safari

vasilev-alex committed:
v0.28.31

lvauvillier committed:
fix(helm-charts): Add global.apiSecretFromSecret value and fix worker deployment naming (#3346)

Commit:
Edit wording

Commit:
Edit wording

Commit:
Merge branch 'helm-chart-improvement' of github.com:cube-js/cube.js into helm-chart-improvement

adnanrahic commented:
Closing in favor of: https://github.com/cube-js/cube.js/pull/3371
